main/tests.py

```diff
@@ -34,28 +34,31 @@ def test_basics(self):
         response = self.c.get("/smoketest/")
         self.assertEqual(response.status_code, 500)
 
-        self.assertIn("FAIL", response.content)
+        self.assertIn("FAIL", response.content.decode('utf-8'))
```

Comment:
I'm not sure about the difference but I solved this with `smart_text`: https://github.com/ccnmtl/django-smoketest/commit/e8ccfa08e29b0e6dd2db12ada5134203fa0fc608  Looks like either way works